tests/pages/services/ServiceExternalVolumes-cy.js

```diff
@@ -0,0 +1,157 @@
+require("../../_support/utils/ServicesUtil");
+
+describe("Services", function() {
```

Comment:
There is A LOT of repetition in this file that perhaps we could re-use some parts, not a blocker too since I expect isolated tests that way making more stable